Uplift/Views/Cells/FacilitiesDropdownCell.swift

```diff
@@ -9,136 +9,96 @@ import UIKit
 
 class FacilitiesDropdownCell: UICollectionViewCell {
 
-    private enum FacilityName: String {
-        case bowlingLanes = "BOWLING LANES"
-        case equipment = "EQUIPMENT"
-        case fitnessCenter = "FITNESS CENTER"
-        case gymnasium = "GYMNASIUM"
-        case swimmingPool = "SWIMMING POOL"
-    }
-
-    private let headerImageView = UIImageView()
-    private let headerNameLabel = UILabel()
-    private let headerOpenLabel = UILabel()
-    private let headerView = DropdownHeaderView(frame: .zero, arrowImage: nil, arrowImageTrailingOffset: 0)
-
+    private let headerView = FacilitiesDropdownHeaderView(frame: .zero)
     private var collectionView: UICollectionView!
-    private var collectionViewHeight: CGFloat = 0
     private var dropdownView: DropdownView!
     private var facility: Facility!
-    private var height: CGFloat = 0
+    static let headerViewHeight: CGFloat = 52
+    static let collectionViewSpacing: CGFloat = 16
+    private var facilitiesIndex: Int!
+    private var headerViewTapped: ((Int) -> ())?
+
+    weak var delegate: GymDetailViewController!
```

Comment:
We should define some `protocol` and make this `delegate` be of that type instead of directly making it of a view controller type.
also change it to `private weak var delegate: SomeDelegate?`